### PR TITLE
Added copy to clipboard function, responsive landing page and implemented routes

### LINKED ISSUE #5: Add react-router-dom
Currently, there are no routes used in the project. Add react-router-dom for different routes like home, code-editor, etc.

### PR DESCRIPTION
**Pull request by Aniket-508**
Closes #5 , #3 , #1

### CODE REVIEW

**Comment by Aniket-508** [comment]
@KshitijDarekar Kindly review this PR.

**Review by KshitijDarekar — COMMENTED** [review]
When a particular editor window is minimized, the copy-to-clipboard icon isn't placed in the same block, it goes in the next Line. Perhaps a display:inline-flex for the div in `line 40` fix that. Well, it isn't a major concern and we can keep it as it is. What do you say?
image









image

**Review by KshitijDarekar — COMMENTED** [review]
Btw, you have done a great job with the landing page. Couldn't expect more, and good use of reusable components.🙌
In the future, we may plan to include support for other languages.
Absolutely, loved the design 🚀

**Comment by KshitijDarekar** [comment]
@Aniket-508 issue[#1 ] was already assigned to someone else. It won't be a problem if the assignee hasn't started working on it already but do check out if any issue is taken up by someone already the next time, to avoid conflicts.

**Comment by Aniket-508** [comment]
@KshitijDarekar I have added inline-flex on the line 40. And regarding the issue 1, the assignee has already given green signal to assign it to someone else. Maybe you can merge this PR then. One more request, can you make the Landing page as difficult since I also made it responsiveness and for all other issues that this PR solves, you can keep them as you want.

File: src/components/Editor.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function Editor(props) {
       <div className={`editor-container ${open ? "" : "collapsed"}`}>
         <div className="editor-title">
           {displayName}
-          <div>
+          <div className="button-group-editor">
```
Comment (KshitijDarekar) [inline src/components/Editor.js:40]: Where is this class `button-group-editor` defined?**

**Comment by KshitijDarekar** [comment]
@Aniket-508 any update?

**Comment by Aniket-508** [comment]
@KshitijDarekar I have already done that. Please read the comments above.

**Comment by KshitijDarekar** [comment]
> @KshitijDarekar I have already done that. Please read the comments above.

you have added the class`button-group-editor` in the last commit, but I can't see the CSS for the same .🤔

**Comment by Aniket-508** [comment]
@KshitijDarekar Extremely sorry, I forgot to stage that file. Now it is done.

**Review by KshitijDarekar — APPROVED** [review]
LGTM

**Pull request merged**